shared/src/packages/orders/libs/validation-schemas/order-create-request-body.validation-schema.ts

```diff
@@ -0,0 +1,21 @@
+import joi from 'joi';
+
+import { positiveRequiredIntegerSchema } from '~/libs/validation-schemas/validation-schemas.js';
+import { UserValidationRule } from '~/packages/users/libs/validation-schemas/enums/enums.js';
+
+import { OrdersValidationMessage } from '../enums/orders-validation-message.enum.js';
+import { type OrderCreateRequestDto } from '../types/types.js';
+
+const orderCreateRequestBody = joi.object<OrderCreateRequestDto, true>({
+  scheduledTime: joi.string().isoDate().required(),
+  carsQty: joi.number().integer().positive().required(),
```

Comment:
can you reuse positiveRequiredInteregerSchema there? do we have some requirements about min/max numbers?

Reply:
done